tokio-codec/src/length_delimited.rs

```diff
@@ -603,8 +607,8 @@ impl Builder {
     /// # Examples
     ///
     /// ```
-    /// # use tokio::io::AsyncRead;
-    /// use tokio::codec::length_delimited::Builder;
+    /// # use tokio_io::AsyncRead;
+    /// use tokio_codec::length_delimited::Builder;
```

Comment:
these examples should probably use `LenthDelimited::builder()`

Reply:
Done